### ISSUE TITLE
Wrong labels of selected classes passed in batch

### ISSUE THREAD

**Issue by ozzyou** [khy0809/fewshot-egnn#13]
While going through your data loader, I noticed that in `fewshot-egnn/data.py` you randomly pick class labels with `task_class_list = random.sample(full_class_list, num_ways)`, but never really pass these labels in your batch. Instead we have

https://github.com/khy0809/fewshot-egnn/blob/205fa80ec7cb12550f7b52a63f921171f92dac4c/data.py#L106

https://github.com/khy0809/fewshot-egnn/blob/205fa80ec7cb12550f7b52a63f921171f92dac4c/data.py#L111

which assigns the wrong class labels ([0, ..., #ways]) to the picked data points. Shouldn't the support as well as the query label be assigned `task_class_list[c_idx]`?

**Comment by jaeho3690** [comment]
yes, I am facing the same issue. I think it should be task_class_list[c_idx]
